### DISCUSSION TITLE
Zip bundle for Windows

### DISCUSSION THREAD

**Discussion by gouravkr** [usebruno/bruno#408] [General]
Would it be possible to create a zip bundle for Windows? It's not always possible to download and run installable exe in a corporate environment (where we often need applications like these). Having a zip bundle would help. 

It will also reduces the overhead of trying it out. With an installer, installation (and subsequent removal if you don't like it) adds some overhead which reduces the number of people who will try it out.

**Comment by helloanoop** [comment]
I just published the portable builds for Win and Mac in `v0.21.0`
You can download the latest build from the [website](https://www.usebruno.com/downloads)

I'm using a Mac.
Can you test it on windows and let me know if it's working as expected.

**Comment by gouravkr** [comment]
Downloaded the zip bundle yesterday. Been trying it out, created collections, post-request script, tests, assertions, etc. Works like charm.
